### PR TITLE
rm: authentication-xilinx: extend integration with LmP

### PR DESCRIPTION
**Pull request by ricardosalveti**
## Readiness

* [x] Merge (pending reviews)
* [ ] Merge after _date or event_
* [ ] Draft

## Overview

Extend instructions to better cover the usage and integration with LmP (when not building outside LmP).

## Checklist

_Optional. Add a 'x' to steps taken._
_You can fill this out after opening the PR. "Did I..."_

* [ ] Run spelling and grammar check, preferably with linter.
* [ ] Avoid changing any header associated with a link/reference.
* [ ] Step through instructions (or ask someone to do so).
* [ ] Review for [wordiness](https://languagetool.org/insights/post/wordiness/)
* [ ] Match tone and style of page/section.
* [ ] Run `make linkcheck`.
* [ ] View HTML in a browser to check rendering.
* [ ] Use [semantic newlines](https://bobheadxi.dev/semantic-line-breaks/).
* [ ] follow best practices for commits.
  * [ ] Descriptive title written in the imperative.
  * [ ] Include brief overview of QA steps taken.
  * [ ] Mention any related issues numbers.
  * [ ] End message with sign off/DCO line (`-s, --signoff`).
  * [ ] Sign commit with your gpg key (`-S, --gpg-sign`).
  * [ ] Squash commits if needed.
* [x] Request PR review by a technical writer and at least one peer.

### CODE REVIEW

**Comment by doanac** [comment]
Docs for 595f7d27b6e75200ca1f8529ffc27ab009eeed1c are browsable at: https://ci.foundries.io/projects/fio-docs/builds/1894/docs/artifacts/html/index.html

**Comment by doanac** [comment]
Docs for 37e18ab1e6a16b227f150d1bc5f067ba735c17c2 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/1895/docs/artifacts/html/index.html

**Comment by caiotpereira** [comment]
LGTM

File: source/reference-manual/security/authentication-xilinx.rst [hunk 1]
```diff
@@ -13,11 +13,19 @@ This is a simple guide on how to provision a device enabling the bootloader hard
...
 Build the bootloader
 --------------------
-Build U-boot for the ZynqMP SoC platform including SPL support ``u-boot-spl.bin`` (also available at the deploy folder if built with LmP).
+Build U-boot for the ZynqMP SoC platform including SPL support, which will produce a file called ``u-boot-spl.bin``.
+
+This file can also be found at the deploy folder when building LmP for secure targets (on secure targets the variable ``SPL_BINARY`` is set to ``spl/u-boot-spl.bin``,
+resulting the publishing of this binary as it assumes that the final ``boot.bin`` will be manually generated and signed by the user).
+
+On normal (open) targets the LmP build process will only publish the final ``boot.bin`` binary, as without any signing requirement the image can
+already be created and published as part of the build process.
```
Comment (kprosise) [inline source/reference-manual/security/authentication-xilinx.rst:28]:
```suggestion
On normal (open) targets, the LmP build process only publishes the final ``boot.bin`` binary.
Without any signing requirements, the image can be created and published as part of the build process.
```
Sentence length

File: source/reference-manual/security/authentication-xilinx.rst [hunk 2]
```diff
@@ -141,6 +149,25 @@ Applying this `patch`_ to U-boot you should see the following on a successful bo
 .. note::
         Booting a secure image disables the JTAG interface even if no JTAG related fuses were written. Use the SPL configuration option `CONFIG_SPL_ZYNQMP_RESTORE_JTAG`_ to re-enable it on boot.
 
+Integrating the signed boot.bin in LmP
```
Comment (kprosise) [inline source/reference-manual/security/authentication-xilinx.rst:152]:
```suggestion
Integrating the Signed boot.bin in LmP
```

**Review by kprosise — APPROVED** [review]
Suggestions made, primarily around sentence length.

**Comment by ricardosalveti** [comment]
Updated, thanks @kprosise !

**Comment by doanac** [comment]
Docs for 07550e3a327fad919b99fc3295757c93dd9416b6 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/1897/docs/artifacts/html/index.html

**Comment by doanac** [comment]
Docs for e6d9372b11464c8bb30cef53571a8ad8e6d4f113 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/1901/docs/artifacts/html/index.html

**Pull request merged**